tests/datastore.py

```diff
@@ -155,21 +153,21 @@
         "openerHash": "8dd01465003a9b1e01c99c904d86aa518b3a5dd9dc8d40fe7d075c726ac073ca",
         "perf": 0
     },
-    "computePlanID": "",
+    "compute_plan_id": "",
     "inModels": None,
     "log": "[00-01-0032-d415995]",
     "objective": {
         "hash": "3d70ab46d710dacb0f48cb42db4874fac14e048a0d415e266aad38c09591ee71",
         "metrics": {
             "hash": "c42dca31fbc2ebb5705643e3bb6ee666bbfd956de13dd03727f825ad8445b4d7",
-            "storageAddress": ""
+            "storage_address": ""
         }
     },
     "outModel": None,
```

Comment:
There are inModels, outModel and outTrunk/HeadModels in this file, It doesn't have much impact but still :)